rasa_core/channels/slack.py

```diff
@@ -135,6 +137,36 @@ def _is_button_reply(slack_event):
     def _get_button_reply(slack_event):
         return json.loads(slack_event['payload'][0])['actions'][0]['name']
 
+    @staticmethod
+    def _sanitize_user_message(text, uids_to_remove):
+        """
```

Comment:
What does this function do? First line should be a one-line summary 
```text
"""One-line summary.

Details here(...)
""

Reply:
thx, will add that